src/index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <main class="page__main">
+      <section class="page__section">
+        <div class="calendar calendar--start-day--sun calendar--month-length--29 page__calendar">
```

Comment:
you could set width style to calendar class, so you didn't need page__calendar class any more